R-package/README.md

```diff
@@ -152,6 +152,9 @@ Rscript build_r.R
 
 The `build_r.R` script builds the package in a temporary directory called `lightgbm_r`. It will destroy and recreate that directory each time you run the script. That script supports the following command-line options:
 
+- `-j[jobs]`: number of threads to use when compiling LightGBM. e.g. `-j4` will try to compile 4 objects at a time.
+    - by default, this script uses single-thread compilation
+    - for best results, set `-j` to the number of physical CPUs
```

Comment:
Also, I believe `by` and `for` should be capitalized, but I'm not sure.
```suggestion
- `-j[jobs]`: number of threads to use when compiling LightGBM. E.g., `-j4` will try to compile 4 objects at a time.
    - by default, this script uses single-thread compilation
    - for best results, set `-j` to the number of physical CPUs
```

Reply:
both are equally correct. The rules for bulleted lists like this are very vague: https://github.com/microsoft/LightGBM/pull/4064#discussion_r593826093

Reply:
Sorry, this time I was confused by a full stop before the list. 😃